specification/2.0/appendix/performance-metrics/README.md

```diff
@@ -0,0 +1,79 @@
+# Runtime Performance Metrics 
+
+The goal of these metrics is to assess the runtime performance needs of glTF assets.  
+By runtime we mean the cpu/gpu, bandwidth and memory resources needed to render the model in realtime.  
+These performance metrics are divided in two major categories - `complexity` and `memory`
+
+## Performance Metric Categories
+
+### Complexity  
+
+The `complexity`category provides metrics to calculate cpu/gpu and memory bandwidth target requirements - larger numbers will mean a more powerful target device is needed.  
+These metrics do not provide a mapping to target device performance, this mapping must be done by users of the metrics.  
```

Comment:
In the complexity category, another interesting metric would be the "depth" of the scene graph.  Runtime systems typically have stack-limits (while pushing the Transformation matrices) and knowing this tree depth is often helpful.
But, this complexity lies at a higher level of abstraction.  It is worth discussing about how/if it would fit in the metric here.

Reply:
Yes, I think this is valid - I will add something along the lines of 'max node depth'

Reply:
Added metric for each scene called maxNodeDepth